src/App.tsx

```diff
@@ -1,10 +1,87 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import * as todosService from './api/todos';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 11105;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [newTitle, setNewTitle] = useState('');
+  const [isSelected, setIsSelected] = useState('');
+
+  const getAllTodo = () => {
+    return (
+      todosService.getTodos(USER_ID).then(serverTodo => {
+        setTodos(serverTodo);
+      })
+    );
+  };
+
+  const addTodo = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    const newTodo = {
+      userId: USER_ID,
+      title: newTitle.trim(),
+      completed: false,
+    };
+
+    todosService.createTodo(newTodo).then(getAllTodo);
```

Comment:
after add new todo no need to get allTodo from server. just update todos state in case success response from server 
something like that
```
todosService.createTodo(newTodo).then(response => setTodos(prevTodos => [ ...prevTodos, response ]))
```
but anyway add/remove todo should be implemented in next task.

![Screenshot from 2023-07-22 16-49-00](https://github.com/mate-academy/react_todo-app-loading-todos/assets/33029864/047e911e-a6d6-43a0-93ca-649572f3be96)